core/plugins/jdo-datanucleus-5/src/main/java/org/apache/isis/core/runtime/system/persistence/PersistenceSession5_Decouple.java

```diff
@@ -98,6 +104,14 @@ public ObjectAdapter adapterFor(
             } catch(ObjectNotFoundException ex) {
                 throw ex; // just rethrow
             } catch(RuntimeException ex) {
+                
+                System.err.println("------------------------------------------");
+                System.err.println("rootOid: "+rootOid.enString());
+                
+                ex.printStackTrace();
+                System.err.println("------------------------------------------");
+
+                
```

Comment:
Add a FIXME so we don't forget to remove.  Or, better, use a LOG.debug(...)

Reply:
fixme added